docs/conventions.md

```diff
@@ -89,6 +89,16 @@ No conventions yet
 - Contract __MUST__ exist only in case existing logic provides for an extension by the user.
 - Contract is a public API and __MUST__ follow [Spryker plugin interfaces specification](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1038092073/INTEGRATED+RFC+Plugin+interface+specification).
 
+## REST API
+- Controller __SHOULD__ have no business logic.
+- Controller __SHOULD__ be named by template of task's id.
+  For task `validation:php:codestyle` - controller name is `ValidationController` and controller's action name is `validationPhpCodestyle` (task's full id in camelCase).
```

Comment:
For example, for the task...
The controller name should be `ValidatePhpCodestyleController`. We need to keep it more specific a bit. 
action name can be `index()` in this case, so the resulting route should be `POST /api/v1/validate-php-codestyle`

Reply:
Fixed